### DISCUSSION TITLE
MAPI_E_FAILONEPROVIDER==0X8004011D

### DISCUSSION THREAD

**Discussion by Linie966** [microsoft/mfcmapi#504] [Q&A]
I get the same error as in #226 with the Version MFCMAPI May 2021 (21.0.21127.03)
Is this also Outlook ADAL bug?
![image](https://user-images.githubusercontent.com/91326655/134677974-8d7dde06-4448-43a4-bd7b-73a45de495fa.png)

![image](https://user-images.githubusercontent.com/91326655/134676880-2875a069-62e6-4949-bdb8-e6c9419a652a.png)

**Comment by KoekieMie** [comment]
#529 registered yesterday, but the solution isn't clear to me
